### PR TITLE
improve css structure and A11y for the footer

### PR DESCRIPTION
**Pull request by ketnek**

### CODE REVIEW

File: css/components/counter.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  width: 120px;
+  border-radius: 10px;
+  background-color: var(--brown);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+}
+
+.counter__display-icon {
```
Comment (t0bbl) [inline css/components/counter.css:19]: just a small thing but shouldnt it be .counter__display--icon?

File: css/components/bookmarks.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/* Bookmarkes */
+
+.bookmarks {}
```
Comment (t0bbl) [inline css/components/bookmarks.css:3]: i do like to leave some classes empty too but our coaches always want me to delete them...

**Review by t0bbl — APPROVED** [review]
looking great!

**Review by gitboej — APPROVED** [review]
great structure, like it, keep it going 👍

**Pull request merged**